source/features/highest-rated-comment.tsx

```diff
@@ -63,6 +95,22 @@ function init(): false | void {
 	}
 }
 
+function getWatchedReactions(): HTMLElement[] {
+	return positiveReactions.flatMap(reaction => select.all(`${COMMENT_SELECTOR} ${reaction}`));
```

Comment:
Nice use of `flatMap` :+1:

Reply:
What's "watched reactions"?

Reply:
Thanks, although I think the method below is better. Not used to select-dom api so didn't know that shortcut.